pymeasure/instruments/eurotest/eurotestHPP120256.py

```diff
@@ -0,0 +1,320 @@
+        This property is a get so will return a string like this:
+        I, RANGE=5000mA, VALUE=1739mA, then measure_current will
+        return a tuple (5000.0, 1739.0) hence the convenience of the get_process.""",
+        get_process=lambda v: (
+            float(re.findall(r'[-+]?([0-9]*\.[0-9]+|[0-9]+)', v[1])[0]),
+            float(re.findall(r'[-+]?([0-9]*\.[0-9]+|[0-9]+)', v[2])[0])
+        )
+    )
+
+    @property
+    def id(self):
+        """ Returns the identification of the instrument """
+        logging.info(f"Requesting instrument identification...")
+
+        response = self.ask("ID")
+        return response.encode(self.response_encoding).decode('utf-8', 'ignore')
+
+    def status(self):
+        """ Returns the unit Status which is a 16bits response where
+        every bit indicates teh state of one subsystem of the HV Source."""
+        logging.info(f"Requesting instrument status...")
+
+        response = self.ask("STATUS,DI")
+        return response.encode(self.response_encoding).decode('utf-8', 'ignore')
+
+    def lam_status(self):
+        """ Returns the LAM status which is the status of the untit from the point
+        of view of the process. Fo example, as a response of asking STATUS,LAM, the HV
+        voltage could response one of the messages from the next list:
+            LAM,ERROR External Inhibit occurred during Kill enable
+            LAM,INHIBIT External Inhibit occurred
+            LAM,TRIP ERROR Software current trip occurred
+            LAM,INPUT ERROR Wrong command received
+            LAM,OK Status OK"""
+        logging.info(f"Requesting instrument LAM status...")
+
+        response = self.ask("STATUS,LAM")
+        return response.encode(self.response_encoding).decode('utf-8', 'ignore')
+
+    def enable_output(self):
+        """ Enables the output of the HV source. """
+        logging.info(f"Enabling the output...")
+
+        self.write("HV,ON")
+        time.sleep(self.COMMAND_DELAY)
+
+    def disable_output(self):
+        """ Disables the output of the HV source. """
+        logging.info(f"Disabling the output...")
+
+        self.write("HV,OFF")
+        time.sleep(self.COMMAND_DELAY)
+
+    def enable_kill(self):
+        """ Enables the kill function of the HV source.
+         When Kill is enabled yellow led is flashing and the output
+         will be shut OFF permanently without ramp if Iout > IOUTmax."""
+        logging.info(f"Configuring instrument kill as enable...")
+
+        self.write("KILL,ENable")
+        time.sleep(self.COMMAND_DELAY)
+
+    def disable_kill(self):
+        """ Disables the kill function of the HV source.
+         When Kill is disabled yellow led is not flashing and the output
+         will NOT be shut OFF permanently if Iout > IOUTmax."""
+        logging.info(f"Configuring instrument kill as disable...")
+
+        self.write("KILL,DISable")
+        time.sleep(self.COMMAND_DELAY)
```

Comment:
It's ok to use methods in these cases, however, you could use a property as well, e.g. output = True/False and kill_mode = True/False (or ON/OFF...)

Reply:
Done in commit. Many thanks.